plans/all.fmf

```diff
@@ -26,6 +26,8 @@ prepare:
 execute:
   how: tmt
   script: |
+    # Testing farm doesn't do a recursive clone
```

Comment:
The tmt run all scripts in plan on a random folder, not in repo folder. We can remove this command to avoid error
```
                    # Testing farm doesn't do a recursive clone
                    git submodule update --init --recursive
                    pip install --user -r test/requirements.txt
                    pytest --force-aws-upload
12:23:01         environment: None
12:23:01                 out: fatal: not a git repository (or any of the parent directories): .git
12:23:01                 out: Shared connection to 10.0.133.33 closed.
12:23:01         Command returned '128'.
```